salt/orchestrate/aws/mitxpro_autoscaling.sls

```diff
@@ -9,7 +9,11 @@
 {% set sqs_queue = env_data.provider_services[app_name].sqs.queue ~ '-' ~ ENVIRONMENT ~ '-autoscaling' %}
 {% set sns_topic = env_data.provider_services[app_name].sns.topic ~ '-' ~ ENVIRONMENT ~ '-autoscaling' %}
 {% set edx_codename = purpose_data.versions.codename %}
-{% set security_groups = purpose_data.get('security_groups', []) %}
+{% if app_name == 'edxapp' %}
+{% set security_groups = 'edx' %}
+{% else %}
+{% set security_groups = 'edx-worker' %}
```

Comment:
Is there a material difference between the settings and uses of these security groups?

Reply:
Yeah there are quite some differences between the two that it's probably best to keep them as is